### PR TITLE
Development

### PR DESCRIPTION
**Pull request by dmamujee**

### CODE REVIEW

File: textDisplay.cc [hunk 1]
```diff
@@ -0,0 +1,82 @@
+#include "textdisplay.h"
+
+TextDisplay::TextDisplay(int n):View(n){
+	// Creates a new nxn textdisplay and sets the default value to '0'
+	theDisplay = new char*[n];
+	for (int row = 0; row < n; row++){
```
Comment (jajsmith) [inline textDisplay.cc:6]: Nice way to assign all the characters! I wonder if we should assign them all " " and "_" and when we load the pieces, then we notify the view that it is changing?

File: textDisplay.cc [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+}
+
+void TextDisplay::notify(int row, int column, char ch){
+ 	theDisplay[row][column] = ch;
+ }
+
+   
+void TextDisplay::print(std::ostream &out) const {
+  	for (int row = 0; row < gridSize; row++){
+  		out << n-row+1 << " ";
```
Comment (jajsmith) [inline textDisplay.cc]: Where does the n come from?

Comment (dmamujee) [inline textDisplay.cc]: the n is a parameter for the ctor, and n is the size of the grid. So this code could be reused for an nxn board. But for now, when we create boards, we just use 8 as the parameter for the ctor

File: textDisplay.cc (hunk 1 above)
Comment (dmamujee) [inline textDisplay.cc:6]: Do you mean initially create a blank board with just "black" and "white" squares, and then notify the view for the initial position for each piece? I like that idea, cause that might be useful for the implementation of Chess960

File: textDisplay.cc (hunk 2 above)
Comment (jajsmith) [inline textDisplay.cc]: That makes sense, but I don't think you've declared n anywhere. Maybe you meant to use gridSize?

File: textDisplay.cc (hunk 1 above)
Comment (jajsmith) [inline textDisplay.cc:6]: Right, when we initialize the game or run "clearGame()" to reset the board, we'll assign the pieces to each cell and then the cells will notify the view.

File: textDisplay.cc (hunk 2 above)
Comment (dmamujee) [inline textDisplay.cc]: True, you're right. Thanks for catching that!

File: textDisplay.cc (hunk 1 above)
Comment (dmamujee) [inline textDisplay.cc:6]: OK sound good, I'll make that change. 
So then for the Load Game feature, should we do that within the game and then just notify the TextDisplay of each piece location in the saved game? Same idea?

**Comment by jajsmith** [comment]
Looks great!

**Pull request merged**